mera/mera/example/mera_pymer4_example.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+
+from mera.mera_pymer4 import run_mera
+
+# Load the data
+data_dir = Path(__file__).parent / "resources"
+output_dir = Path(__file__).parent / "residuals"
+output_dir.mkdir(exist_ok=True)
+
+stations_ffp = data_dir / "stations.csv"
+events_ffp = data_dir / "events.csv"
+obs_ffp = data_dir / "im_obs.csv"
+sim_ffp = data_dir / "im_sim.csv"
+
+stations_df = pd.read_csv(stations_ffp, index_col=0)
+events_df = pd.read_csv(events_ffp, index_col=0)
+
+obs_df = pd.read_csv(obs_ffp, index_col=0)
+sim_df = pd.read_csv(sim_ffp, index_col=0)
+
+# Sanity checking
+# Ensure the index matches
+assert (
+    np.all(obs_df.index == sim_df.index)
+    and np.all(obs_df.event_id == sim_df.event_id)
+    and np.all(obs_df.stat_id == sim_df.stat_id)
+)
+
+# List of IMs of interest
+# Just using all shared pSA periods in the data files
+ims = [
+    cur_im
+    for cur_im in np.intersect1d(obs_df.columns, sim_df.columns)
+    if cur_im.startswith("pSA")
+]
+
+# Compute the residual
+res_df = (obs_df[ims] / sim_df[ims]).apply(np.log)
```

Comment:
You didn't just do "np.log(obs_df/sim_df)"?
Is there much of a difference?

Reply:
No they are identical, for some reason I thought doing np.log directly returns a numpy array. No idea why since it doesn't really make sense. Will change it